src/services/IngestorService.ts

```diff
@@ -40,15 +40,29 @@ export class IngestorService {
         throw constants.EMPTY_DATASET_ID
     }
 
+    public async getDatasetConfig(datasetId: string) {
+        let datasetConfigList = globalCache.get("dataset-config");
```

Comment:
Why this change here?

Reply:
Have refactored to cache the datasets, as the old function was querying the db on each request